src/TesApi.Web/BatchScheduler.cs

```diff
@@ -56,7 +55,7 @@ public partial class BatchScheduler : IBatchScheduler
         private const string StartTaskScriptFilename = "start-task.sh";
         private const string NodeTaskRunnerFilename = "tes-runner";
         private const string NodeTaskRunnerMD5HashFilename = NodeTaskRunnerFilename + ".md5";
-        private readonly string cromwellDrsLocalizerImageName;
+        // private readonly string cromwellDrsLocalizerImageName;
```

Comment:
lets just delete this and all associated code

Reply:
Janet has stated that this is still required.